descope/internal/mgmt/accesskey.go

```diff
@@ -89,17 +92,20 @@ func (a *accessKey) Delete(ctx context.Context, id string) error {
 	return err
 }
 
-func makeCreateAccessKeyBody(name string, expireTime int64, roleNames []string, tenants []*descope.AssociatedTenant, userID string, customClaims map[string]any, description string, permittedIPs []string) map[string]any {
-	return map[string]any{
+func makeCreateAccessKeyBody(name string, expireTime int64, roleNames []string, tenants []*descope.AssociatedTenant, userID string, customClaims map[string]any, description *string, permittedIPs []string) map[string]any {
+	res := map[string]any{
 		"name":         name,
 		"expireTime":   expireTime,
 		"roleNames":    roleNames,
 		"keyTenants":   makeAssociatedTenantList(tenants),
 		"userId":       userID,
 		"customClaims": customClaims,
-		"description":  description,
 		"permittedIps": permittedIPs,
 	}
+	if description != nil {
+		res["description"] = description
+	}
+	return res
```

Comment:
i don't think it makes sense to use a pointer string in the creation since if not provided, an empty description will be used, right?